geometric/tests/test_qcengine.py

```diff
@@ -41,3 +42,81 @@ def test_rdkit_simple(localizer):
     # Currently in angstrom
     ref = np.array([0., 0., -0.0644928042, 0., -0.7830365196, 0.5416895554, 0., 0.7830365196, 0.5416895554])
     assert np.allclose(ref, ret.xyzs[-1].ravel(), atol=1.e-5)
+
+
+@addons.using_qcengine_genopt
+@pytest.mark.parametrize("bsse", [
+    pytest.param("nocp"),
+    pytest.param("cp"),
+])
+@pytest.mark.parametrize("qcprog", [
+    # different programs just for flexibility of testing
+    pytest.param("psi4", marks=addons.using_psi4),
+    pytest.param("cfour", marks=addons.using_cfour),
+    pytest.param("nwchem", marks=addons.using_nwchem),
```

Comment:
I don't think geomeTRIC has a NWChem interface.

Reply:
Right, integrating QCManyBody for geomeTRIC's first-class engines is a separate project (and one I didn't even think of initially). This PR is purely so that `geometric.run_json` can take the slightly different schema layout for an OptimizationInput holding an ordinary gradient specification vs. holding a many-body gradient specification. (The divergence isn't long-term; we're revising the schema.) So the PR is really only good for the run_json half of the QCEngine engine of geomeTRIC.